### ISSUE TITLE
Linking to an R6 method

### ISSUE THREAD

**Issue by hongooi73** [r-lib/roxygen2#1039]
Hello all,

See example below. I want to refer to method `g`, both in the documentation for method `f` and the standalone function `h`. Is there an easy way of doing this?

```r
#' A class
#' @description
#' Description of a class.
newclass <- R6::R6Class("newclass", public=list(

  #' @description
  #' Here is a link to to the documentation for g: [newclass$g]  # ???
  f=function() {},

  #' @description
  #' something relevant to f, g and h
  g=function() {}
))

#' A function
#' @description
#' Description of a standalone function.
#' @seealso
#' [newclass$g]  # ???
h <- function() {}
```

**Comment by Hong-Revo** [comment]
A workaround which I found in [pkgdepends](https://github.com/r-lib/pkgdepends) is to link to an anchor:

```r
#' A class
#' @description
#' Description of a class.
newclass <- R6::R6Class("newclass", public=list(

  #' @description
  #' Here is a link to to the documentation for g: [`g`](#method-g)
  f=function() {},

  #' @description
  #' something relevant to f, g and h
  g=function() {}
))

#' A function
#' @description
#' Description of a standalone function.
#' @seealso
#' [`newclass$g`][newclass#method-g]
h <- function() {}
```

However, if you try to link to a method in another Rd file (like in the block for `h` above), this throws a warning in R CMD check. It still works though (at least for HTML help).

```
W  checking Rd cross-references (411ms)
   Missing link or links in documentation object 'QueueMessage.Rd':
     'StorageQueue#method-peek_message' 'StorageQueue#method-get_messages'

   See section 'Cross-references' in the 'Writing R Extensions' manual.
```

**Comment by gaborcsardi** [comment]
Proper linking should be solved with #955.
